src/FsAutoComplete.Core/AdaptiveExtensions.fs

```diff
@@ -699,39 +748,48 @@ module AsyncAVal =
           if x.OutOfDate then
             if Interlocked.Exchange(&inputChanged, 0) = 1 || Option.isNone cache then
               let outerTask =
-                RefCountingTaskCreator(
-                  cancellableTask {
-                    let! i = value.GetValue t
-                    let! ct = CancellableTask.getCancellationToken ()
-                    let inner = mapping i ct
-                    return inner
+                RefCountingTaskCreator(fun ct ->
```

Comment:
is this the same throughout this file? moving from `cancellableTask` to `task` for better stack traces?

Reply:
Yeah I need to rundown why I was still getting weird stacktraces but I'll do that later.